clar2wasm/tests/standard/unit_tests.rs

```diff
@@ -1826,3 +1826,297 @@ fn bit_not_int() {
     assert_eq!(result[0].i64(), Some(-4));
     assert_eq!(result[1].i64(), Some(-1));
 }
+
+#[test]
+fn pow_uint() {
+    let (instance, mut store) = load_stdlib().unwrap();
+    let pow = instance.get_func(&mut store, "pow-uint").unwrap();
+    let mut result = [Val::I64(0), Val::I64(0)];
+
+    // pow(0, 0) == 1
+    pow.call(
+        &mut store,
+        &[Val::I64(0), Val::I64(0), Val::I64(0), Val::I64(0)],
+        &mut result,
+    )
+    .expect("call to pow-uint failed");
+    assert_eq!(result[0].unwrap_i64(), 1);
+    assert_eq!(result[1].unwrap_i64(), 0);
+
+    // pow(0, 0) == 1
+    pow.call(
+        &mut store,
+        &[Val::I64(0), Val::I64(0), Val::I64(0), Val::I64(0)],
+        &mut result,
+    )
+    .expect("call to pow-uint failed");
+    assert_eq!(result[0].unwrap_i64(), 1);
+    assert_eq!(result[1].unwrap_i64(), 0);
```

Comment:
Is this the same as the test above?

Reply:
Abosolutely. We are never too sure... I guess :D

Reply:
fixed in ac6aaa5838fcd3e82357323d4619b637521872c3